bool is_activated
string command